main.go

```diff
@@ -43,7 +51,15 @@ func main() {
 	}
 	// Kick off the callback client, to send job update messages on a regular interval
 	statusClient := clients.NewPeriodicCallbackClient(15 * time.Second).Start()
-	vodEngine, err := pipeline.NewCoordinator(pipeline.Strategy(*vodPipelineStrategy), mist, *externalTranscoderUrl, statusClient)
+	if metricsDBConnectionString != nil && *metricsDBConnectionString != "" {
+		metricsDB, err = sql.Open("postgres", *metricsDBConnectionString)
+		if err != nil {
+			log.Println("Error creating postgres metrics connection: ", err)
```

Comment:
I think this should probably fail to start, since the user was trying to run it with metrics enabled